### DISCUSSION TITLE
Audio export for scenes not working

### DISCUSSION THREAD

**Discussion by Pixsoll** [keiwando/toonsquid#76] [Q&A]
For one of my projects, whenever I export it, only the first scene has the audio while the other scenes are muted. This doesn’t happen with any of my other projects and when in the program, I can hear the audio for all the scenes perfectly fine. Does anyone know what’s happening?

**Comment by keiwando** [comment]
Hi. Thank you very much for reporting this bug! I've identified the issue and a fix will be part of the next update.
Sorry about that!
